spec/migrations/20220513071902_update_service_features_spec.rb

```diff
@@ -0,0 +1,41 @@
+require_migration
+
+describe UpdateServiceFeatures do
+  let(:user_role_id) { id_in_current_region(1) }
+  let(:feature_stub) { migration_stub :MiqProductFeature }
+  let(:roles_feature_stub) { migration_stub :MiqRolesFeature }
+
+  migration_context :up do
+    describe 'product features renaming' do
+      it 'renames the Services features' do
+        UpdateServiceFeatures::FEATURE_MAPPING.each_key do |identifier|
+          feature = feature_stub.create!(:identifier => identifier)
+          roles_feature_stub.create!(:miq_product_feature_id => feature.id, :miq_user_role_id => user_role_id)
+        end
```

Comment:
I'm having trouble reading this dynamic code.
Can you just explicitly type in the feature values for the before and after?